### PR TITLE
Feature/gender published data

### PR DESCRIPTION
**Pull request by roll**
https://github.com/opentrials/opentrials/issues/84

### CODE REVIEW

**Comment by roll** [comment]
@vitorbaptista please review

File: processors/gsk/extractors.py [hunk 1]
```diff
@@ -19,9 +19,22 @@ def extract_source(record):
...
+
     # TODO: fix
+    # Get registration date
     registration_date = (
         record['first_received'] or datetime.datetime.now().date())
+
+    # Get gender
+    gender = None
+    if record['gender'] in ['Both', 'Male', 'Female']:
+        gender = record['gender'].lower()
```
Comment (vitorbaptista) [inline processors/gsk/extractors.py:31]: When a gender can be different than "Both", "Male" or "Female"? It would be good to add a small comment here.

**Comment by vitorbaptista** [comment]
@roll Looks good :+1:

Just a suggestion, you use this pattern in many places:

``` python
foo = False
if bar:
    foo = True
```

That could be simplified to:

``` python
foo = bool(bar)
```

Or, when the condition is an actual test, you could do:

``` python
foo = (bar == "baz")
```

If the condition is small, I think it's cleaner. However, that's just a minor code style change. Feel free to merge the code as is.

**Comment by roll** [comment]
I've removed gender checks - let fail (process will be ok it's only one iteration and it will be in Papertrial) if something will change in warehouse. So I'll be notified about a problem.

About conditions:

```
foo = False
if bar:
    foo = True
```

I prefer this style because it's more like pseudo-language (could be understood by anyone) and more importantly there is no if branch hiding so code coverage tool will flag about untested condition (almost not applicable here of course).

**Pull request merged**